### PR TITLE
Update introduction.rst

### PR DESCRIPTION
**Pull request by jbcrestot**
Just a lil typo

### CODE REVIEW

File: reference/forms/types/radio.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@
 Type de champ Radio
 ===================
 
-Crée un simple bouton radio. Il devrait toujours être utilisé pour un champ
-dont la valeur est booléenne. : si le bouton radio est sélectionné, le champ sera
-défini à true, sinon, la valeur sera définie à false.
+Créé un unique bouton radio. Si le bouton radio est sélectionné, le champ aura pour valeur
```
Comment (gregquat) [inline reference/forms/types/radio.rst]: "Crée" plutôt que "créé"

**Comment by gregquat** [comment]
Thank you. I'll merge it once you have fixed the very little typo ;)

**Comment by jbcrestot** [comment]
Done, Thanks to github éditor online. :)

**Comment by gregquat** [comment]
And thanks for the quick fix!

**Pull request merged**